Framework/Geometry/src/RandomPoint.cpp

```diff
@@ -51,16 +78,33 @@ Kernel::V3D inCylinder(const detail::ShapeInfo &shapeInfo,
   const double z{geometry.height * r3};
   const Kernel::V3D alongAxis{geometry.axis * z};
   const Kernel::V3D &basis1{geometry.axis};
-  Mantid::Kernel::V3D basis2{1., 0., 0.};
```

Comment:
With the removal of this code from `inCylinder()`, the `using boost::math::pow` at the beginning of the function has lost its meaning and can be removed.